rfcs/0136-stabilize-incrementally.md

```diff
@@ -0,0 +1,199 @@
+This "dusty corner" off the new CLI is rather calm with a very constrained design space, and far less acrimony.
+
+## Conclusion
+
+By starting with these relatively easy material to stabilize, we can prove we can all come together as a community to agree on a design after all.
+This should build trust between opposing individuals and factions, giving us a foundation upon which to tackle the more challenging material in subsequent steps.
+
+Since these features became very popular while they are still unstable, there is now both an urgency to stabilize them, and little ability to modify them.
+This is the opposite of how experimental features are supposed to leisurely incubate with both plenty of flexibility to change them, and little rush to stabilize them until they feel ready.
+But, there is little we can do about this at this point, an this RFC recognizes that fact and does *not* try to resist it.
+
+But, we have crossed that Rubicon and there is no turning back; this RFC *doesn't* attempt to change Flakes or the command line 
+
+# Detailed design
+[design]: #detailed-design
+
+Step 1 is technical work, with a self-imposed deadline so we can be sure it doesn't delay stabilization too long.
+The remaining steps are stabilization steps.
+For each of them, a separate RFC or other discussion medium will describe the new interfaces to be stabilized, and solicit feedback.
+
+## Step 0: Stabilize the store-only installable-free CLI
+
+There are certain commands like `nix store gc`, `nix store ping` that do not take any positional arguments.
+As @roberth elsewhere pointed out, because these commands have so few degrees of freedom, they are some of the easiest to stabilize --- there is simply less to pour over and possibly bikeshed.
+
+We can start stabilizing them right away, either in a batch, or one by one.
+
+## Step 1: Split out a store-only Nix, with deadline
+
+This is the same as step one from [RFC 134](https://github.com/NixOS/rfcs/pull/134), upon which this RFC depends.
+
+A deadline is imposed to get that complete, so agreement on this stabilization plan is guaranteed not to add more than a small bounded delay as opposed to stabilizing everything immediately.
+
+## Step 2: Stabilize the store-only Nix CLI
+
+If the deadline for step 1 is met, then we stabilize *just* the command-line interface of the store-only Nix command.
+
+This is a small portion of the total amount of interface we have to stabilize, and that will hopefully yield a narrow and more focused discussion.
+Yet it will still offer some interesting topics to discus, such as:
+
+- logging
+- store paths on `stdout` at end of build?
+- Should commands like `show-derivation` should use `--json` by default
+- Flat vs hierarchical commands
+- is `--derivation` a good flag?
+
+## Step 3: Stabilize the rest of the CLI, without Flakes
+
+If the deadline for step 1 is *not* met, we jump directly to this step, skipping step 2.
+
+This is rest off the new CLI, without flakes.
```

Comment:
> This is rest off the new CLI, without flakes.

I can't really parse this sentence. Should I read it as "This is **the** rest of the new CLI"?

Reply:
```suggestion
This is the rest off the new CLI, without flakes.
```